aggregators/1.0/modules/ROOT/pages/aggregators-size-example.adoc

```diff
@@ -0,0 +1,140 @@
+= Size-Based Aggregator Example - Mule 4
+
+The Aggregators module *Size based aggregator* scope enables you to aggregate elements until a pre-defined size number of elements completes the aggregation. The following example shows how to configure this scope to aggregate up to `3` HTTP request calls. Then, the application transforms the calls via DataWeave into random ID numbers. When the aggregation reaches `3` calls, the application logs all the aggregated random ID number elements.
```

Comment:
> Then, the application transforms the calls via DataWeave into random ID numbers.

This confuses me a bit. The idea is to set a payload with an ID to verify later that it is the same payload that was aggregated. Since we don't care about the content, this ID just needs to be different each time, that's why a random number is used.